packages/geo/src/coordinates/latlon/internal/lexer.ts

```diff
@@ -0,0 +1,61 @@
+// __private-exports
+/*
+ * Copyright 2024 Hypergiant Galactic Systems Inc. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import { Patterning } from '@/patterning';
+
+import { SYMBOL_PATTERNS, Symbols } from '.';
+
+export type Tokens = ReturnType<typeof lexer>;
+
+const patterns = {
+  positive: /\+/g,
+  separators: /[,/]/g,
```

Comment:
We may want to leave a note here or future TODO to support a space as a separator (will sanity check if that is a common scenario in the field first)